### PR TITLE
Add 5.19 release notes

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+
+### 2 New Cops
+
+#### ChefRedundantCode/GroupingMetadata
+
+The `ChefRedundantCode/GroupingMetadata` cop removes the legacy `grouping` metadata from the metadata.rb file. This metadata was never consumed by any Chef Infra services and does not need to be defined.
+
+#### ChefStyle/OverlyComplexSupportsDependsMetadata
+
+The `ChefStyle/OverlyComplexSupportsDependsMetadata` cop cleans up overly complex methods of declaring `supports` or `depends` metadata via an array. This rule will detect and correct the array/each form of declaring this metadata when there are fewer than three items.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
The `ChefStyle/OverlyComplexSupportsDependsMetadata` cop cleans up overly complex methods of declaring `supports` or `depends` metadata via an array. This rule will detect and correct the array or each form of declaring this metadata when there are fewer than three items.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+
+```ruby
+depends 'apt'
+depends 'windows'
+```
+
+### Other fixes and changes
+
+- Various cops that remove legacy code no longer leave behind empty lines when autocorrecting.
+- `ChefRedundantCode/LongDescriptionMetadata` now properly autocorrects when `long_description` uses a HEREDOC.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
- `ChefRedundantCode/LongDescriptionMetadata` now properly autocorrects when `long_description` uses a here document (heredoc).
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+- `ChefDeprecations/UserDeprecatedSupportsProperty` no longer fails to autocorrect when the supports hash uses Ruby 1.8 has rocket syntax.
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
- `ChefDeprecations/UserDeprecatedSupportsProperty` no longer fails to autocorrect when the supports hash uses Ruby 1.8 and has rocket syntax.
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+- `ChefStyle/FileMode` now autocorrects file modes using single quotes instead of double quotes.
+- `ChefSharing/InvalidLicenseString` now autocorrects `UNLICENSED` to `all rights reserved`
```
Comment (IanMadd) [inline RELEASE_NOTES.md]:
```suggestion
- `ChefSharing/InvalidLicenseString` now autocorrects `UNLICENSED` to `all rights reserved`.
```

File: RELEASE_NOTES.md (hunk 3 above)
Comment (tas50) [reply RELEASE_NOTES.md]: This is actually a typo. I'll fix it. It should be Ruby 1.8 hash rocket syntax

**Pull request merged**